contracts/0xcert/ERC165.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.23;
+
+/**
+ * @dev A standard for detecting smart contract interfaces. See https://goo.gl/cxQCse.
```

Comment:
link the EIP directly instead of through google: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md

Reply:
I figured this whole file was copied from 0xcert, including these comments

Reply:
that's the beauty of the MIT license! we have freedom to modify the code as much as we want